bots/synclabel/src/main/java/org/openjdk/skara/bots/synclabel/SyncLabelBotCheckWorkItem.java

```diff
@@ -32,12 +32,12 @@
 
 public class SyncLabelBotCheckWorkItem implements WorkItem {
```

Comment:
Now that it is not only checking, but also updating the labels, maybe rename the class?

Reply:
Makes sense, will change it.